src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -18,19 +18,19 @@ export const AccordionItem = ({id, label, icon, onClick, children, className, ..
         <section
             {...props}
             className={classnames(
-                styles.accordionItem,
-                {[styles.reversed]: context.isReversed},
+                'moonstone-accordionItem',
+                {reversed: context.isReversed},
```

Comment:
shouldn't we prefix all classes, including modifiers classes ?

Reply:
I didn't add the prefix to the modifier classes because they require the top level 'moonstone-*' to be present as well, so it's already specific enough. Although, do you think that reasoning would be confusing for other people creating components?

E.g., 'reversed' is always used with 'moonstone-accordionItem' or 'moonstone-accordionItem_header' (https://github.com/Jahia/moonstone/blob/scss-build/src/components/Accordion/AccordionItem/AccordionItem.scss#L19)

Reply:
I was only thinking of prefixing them, because if somebody is also defining classes with same names (like "reversed"), we may get some styles from here which are not wanted .. for the opposite I agree that somebody using "reversed" will not get moonstone styles since it require the top-level class. Still I'm not fond of prefixing all classes, but there's a risk.

Reply:
Ok! I will rename the all the modifier classes (and any other classes that aren't already prefixed) with the moonstone- prefix.